.github/workflows/neur1630-fall-tag.yml

```diff
@@ -0,0 +1,52 @@
+name: tag-image-neur1630
+on: [workflow_dispatch]
+
+env:
+  CLASS: neur1630
+  TARGET: base
+  SQLITE: false
+  PYTHON_VERSION: "3.9.17"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.3
```

Comment:
This doesn't really matter for this time around, but I think we can probably just use `checkout@v3` and the system will pick the latest minor version? Saves effort for some future updates :)